chat-commands.js

```diff
@@ -2196,7 +2196,19 @@ const commands = {
 			return this.errorReply(`The note is too long. It cannot exceed ${MAX_REASON_LENGTH} characters.`);
 		}
 		if (!this.can('receiveauthmessages', null, room)) return false;
-		this.modlog('NOTE', null, target);
+		if (target.startsWith('\n')) target = target.slice(1);
+		const separator = '\n';
+		if (target.includes(separator) || target.length > 150) {
+			const params = target.split(separator);
+			let output = [];
+			for (const param of params) {
+				output.push(Chat.escapeHTML(param));
+			}
+			let code = `${output.join('; ')}`;
```

Comment:
???

Reply:
I reused this part from the /code command. It joins newlines with a semicolon and a space.

Reply:
I would do it this way. 
```
let output = '';
for (const param of params) {
   output += output.length ? '; ' : '' + Chat.escapeHTML(param);
}
this.modlog('NOTE', null, output);
```

Reply:
Attempting to use it this way seems to have hidden the additional lines in the modlog, only showing "1; ; " instead of "1; 2; 3"